services/core-api/app/api/mines/permits/permit_conditions/models/permit_condition_category.py

```diff
@@ -21,6 +21,13 @@ class PermitConditionCategory(SoftDeleteMixin, AuditMixin, Base):
     active_ind = db.Column(db.Boolean, nullable=False, server_default=FetchedValue())
     display_order = db.Column(db.Integer, nullable=False)
 
+    user_sub = db.Column(db.String, db.ForeignKey('user.sub', ondelete='SET NULL', onupdate="CASCADE"))
+
+    assigned_review_user = db.relationship(
+        'User',
+        back_populates='permit_condition_categories'
```

Comment:
I'm wondering if we should maybe make this a joined, or selectin query to avoid having this as a separate query for each permit condition category (lazy='joined' or lazy='selectin')

Reply:
Sure, that makes sense!